tests/behavioural/features/message_post.feature

```diff
@@ -55,6 +55,7 @@ Feature: Message Send Endpoint
     When the message is sent
     Then a 400 error status is returned
 
-
-
-
+  Scenario: When a message with the label of "Draft" is sent and another user is trying to send the same message return a 409
```

Comment:
We also need a test so that the etag is NOT checked in this same scenario if the POST is done without an etag